weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTable.tsx

```diff
@@ -81,7 +82,7 @@ import {ManageColumnsButton} from './ManageColumnsButton';
 
 const OP_FILTER_GROUP_HEADER = 'Op';
 const MAX_EVAL_COMPARISONS = 5;
-const MAX_BULK_DELETE = 10;
+const MAX_SELECT = 100;
```

Comment:
we support up to 100 in the backend, and most people who delete more than 5 calls delete 10, presumably they are hitting this max. Easy to revert if this causes problems.